src/main/java/openfasttrack/cli/CommandLineInterpreter.java

```diff
@@ -73,8 +76,11 @@ public CommandLineInterpreter(final String[] arguments, final Object argumentsRe
     {
         final Class<?> receiverClass = argumentsReceiver.getClass();
         final Stream<Method> methods = Arrays.stream(receiverClass.getMethods());
-        return methods.filter(method -> method.getName().startsWith(SETTER_PREFIX)) //
+        final Map<String, Method> setterMethods = methods
+                .filter(method -> method.getName().startsWith(SETTER_PREFIX)) //
                 .collect(toMap(CommandLineInterpreter::getSetterName, Function.identity()));
+        LOG.finest(() -> "Found arguments " + setterMethods.keySet());
```

Comment:
Please rephrase. Log message sounds like parameter in the actual command entered by the user was found.